changes/changes.rst.j2

```diff
@@ -0,0 +1,77 @@
+{#
+ # Jinja2 template for towncrier to generate the change log from change fragments.
+ #
+ # Input variables (as of towncrier 22.8.0):
+ # - render_title (bool): Indicates whether a title should be rendered by the
+ #   template. If False, the title is being added by towncrier to the result of
+ #   the template.
+ # - sections: The change fragments, as a dict with key: section
```

Comment:
This is really hard to read, but I assume it gets rendered into sth more human readable?

Reply:
I see ... but I guess there isn't a way to describe these nested types with pure text in a much better way, so be it.